apis/quay/v1/quayregistry_types.go

```diff
@@ -37,16 +37,16 @@ var QuayVersionCurrent QuayVersion = QuayVersion(os.Getenv("QUAY_VERSION"))
 type ComponentKind string
 
 const (
-	ComponentBase          ComponentKind = "base"
-	ComponentPostgres                    = "postgres"
-	ComponentClair                       = "clair"
-	ComponentRedis                       = "redis"
-	ComponentHPA                         = "horizontalpodautoscaler"
-	ComponentObjectStorage               = "objectstorage"
-	ComponentRoute                       = "route"
-	ComponentMirror                      = "mirror"
-	ComponentMonitoring                  = "monitoring"
-	ComponentTLS                         = "tls"
+	ComponentBase          = "base"
```

Comment:
I would keep these constants typed.

Reply:
I think you gonna have to add `ComponentKind` to each line, otherwise only the first one is of type `ComponentKind`. Can you do that?